src/components/tooltip/tooltip.test.ts

```diff
@@ -134,4 +134,17 @@ describe('<sl-tooltip>', () => {
     expect(afterHideHandler).to.have.been.calledOnce;
     expect(base.hidden).to.be.true;
   });
+
+  it('should recalculate its target when the slotted element changes', async () => {
+    const el = await fixture<SlTooltip>(html`
+      <sl-tooltip content="This is a tooltip" open>
+        <sl-button>Hover me</sl-button>
+      </sl-tooltip>
+    `);
+
+    el.innerHTML = '<sl-button>New element</sl-button>';
+    await el.updateComplete;
+
+    expect((el as unknown as { target: HTMLElement }).target.innerHTML).to.equal('New element');
```

Comment:
This tests a private property, which is probably bad (and means I had to do some type shenanigans). Would love a suggestion cause I don't know how to test "accurate positioning of the tooltip".

Reply:
While `target` is private, `getTarget()` is not so this should work fine.
```suggestion
    expect(el.getTarget().textContent).to.equal('New element');
```

Reply:
`getTarget()` does recalculate the target though, so this would pass even without the slotchange addition.

Reply:
Ugh, good call. I've reverted it to this:

```ts
// @ts-expect-error - target is a private property
expect(el.target.innerHTML).to.equal('New element');
```

This will break if we switch to actual `#private` properties, but we'll cross that bridge when we get to it.